### PR TITLE
Add item header

### LINKED ISSUE #106: Item details: header
Start the item page with (according to designs)
- the title
- the subtitle (if any)
- a tab "content" with a "description" section if type === 'Chapter'

### PR DESCRIPTION
**Pull request by GabrielVidal1**
Fixes #106 
Add item header component.

### CODE REVIEW

File: src/app/modules/item/http-services/get-item-by-id.service.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export interface Item {
   requires_explicit_entry: boolean
   string: {
     title: string|null,
+    subtitle: string|null,
```
Comment (smadbe) [inline src/app/modules/item/http-services/get-item-by-id.service.ts:11]: Subtitle is both nullable and omitable (cfr API, "nullable" and not required), so it is `subtitle?: string|null`

**Review by smadbe — CHANGES_REQUESTED** [review]

File: src/app/modules/item/components/item-header/item-header.component.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="task-header-container">
+  <div class="task-header">
+    <div class="task-user-image">
+      <img src="assets/images/_messi.jpg" style="width: 4rem; height: 4rem;" />
```
Comment (smadbe) [inline src/app/modules/item/components/item-header/item-header.component.html:4]: ah and don't put an image of messi by default ;-)

Comment (GabrielVidal1) [reply src/app/modules/item/components/item-header/item-header.component.html]: fixed, I removed the image all together

File: src/app/modules/item/http-services/get-item-by-id.service.ts (hunk 1 above)
Comment (GabrielVidal1) [reply src/app/modules/item/http-services/get-item-by-id.service.ts]: fixed

**Review by smadbe — APPROVED** [review]

**Pull request merged**